conode/run_conode.sh

```diff
@@ -0,0 +1,345 @@
+#!/usr/bin/env bash
```

Comment:
Is there no way to use the one defined in onet or cothority somewhere ? Instead of duplicating the code.